src/drawtypes/label.cpp

```diff
@@ -221,6 +221,17 @@ namespace drawtypes {
       }
     }
 
+    size_t maxlen = conf.get(section, name + "-maxlen", 0_z);
+    bool ellipsis = conf.get(section, name + "-ellipsis", true);
+
+    if(ellipsis && maxlen > 0 && maxlen < 3) {
+      logger::make().err(sstream() << "Label " << section << "." << name
```

Comment:
Couldn't we throw an `application_error` or some other type of error to avoid creating a new logger?

Reply:
`logger::make` uses the singleton pattern, so only the first call to `logger::make()` will create a new instance, all others always return the same instance. I used it directly because I didn't want to add the `m_log` field just for that.. Btw. I really don't like the use of a singleton for the logger but that is a discussion for another time.

If we threw an `application_error` here, wouldn't it be caught in the controller and would disable the whole module? I was debating, if we wanted invalid maxlen be a fatal error, would you prefer that?

Reply:
Yeah, I think that would be much neater.

Reply:
Alright, gonna revert back to throwing an error when I have the time

Reply:
Done